### ISSUE TITLE
Get Some Error

### ISSUE THREAD

**Issue by SherlockHolmes221** [yoyomimi/AS-Net#16]
running the test code on HICO-DET  python3 tools/eval.py --cfg configs/hico.yaml
![088935aaa1b5648f58d7b80a537bba1](https://user-images.githubusercontent.com/28208801/134835939-34c8d91a-f0f4-40dc-a65a-dacf04b89e8d.png)

**Comment by SherlockHolmes221** [comment]
More question: Could you provide the training log files? How many GPUs and the batch sizes did you use to get the final results in paper? Thanks

**Comment by yoyomimi** [comment]
Please check the closed issue to see if it works for you: [issues_7](https://github.com/yoyomimi/AS-Net/issues/7).
Sorry we currently don't have the training logs. We train with 32 GPUs and 2 sample for each GPU. You can decrease the GPU amounts according to your devices, and then modify the learning rate correspondingly, e.g.: 32 GPU * 2 img/per GPU(batchsize 64) & lr -> 8 GPU * 2 img/per GPU(batchsize 16) & 0.25lr.

**Comment by SherlockHolmes221** [comment]
Thanks
